javascript.md

```diff
@@ -33,6 +33,8 @@ JavaScript CodeStyle
   - [Function Context](#function-context)
   - [Comments](#comments)
   - [Classes](#classes)
+  - [ECMAScript 6](#ecmascript-6)
+    - [Variable declaration](#variable-declaration-1)
```

Comment:
I want to describe es6 style in separate section, because most of our code are written in es5. This section extends main section for es5.